fedora/client/openidbaseclient.py

```diff
@@ -154,6 +160,20 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
 
         # python-requests session.  Holds onto cookies
         self._session = requests.session()
+
+        # Also hold on to retry logic.
+        # http://www.coglib.com/~icordasc/blog/2014/12/retries-in-requests.html
+        if retries is not None:
+            prefixes = ['http://', 'https://']
+            for prefix in prefixes:
+                self._session.mount(prefix, requests.adapters.HTTPAdapter(
+                    max_retries=Retry(
+                        total=retries,
+                        status_forcelist=[500],
```

Comment:
Does this mean the request will be retried just for 500 status code? We see Bodhi returning other 5xx codes quite frequently. From a quick search I see a number of 503 a 504 codes. I suspect there are others as well. Could this be changed to retry any 5xx status code? Or do you want to list just the ones we know about and add to the list in the future? I have only experience with Bodhi, but different services might send different error codes.